Signal/src/call/SignalCall.swift

```diff
@@ -17,6 +17,10 @@ enum CallState: String {
     case remoteBusy // terminal
 }
 
+enum CallDirection {
+    case outgoing, incoming
+}
```

Comment:
Hmm, I wonder if its worth trying to DRY this up with the enum in PCC.  Maybe not.